exporters/prometheus/exporter.go

```diff
@@ -205,8 +206,15 @@ func (c *collector) Collect(ch chan<- prometheus.Metric) {
 
 			ch <- scopeInfo
 
-			kv.keys = append(kv.keys, scopeNameLabel, scopeVersionLabel)
-			kv.vals = append(kv.vals, scopeMetrics.Scope.Name, scopeMetrics.Scope.Version)
+			kv.keys = append(kv.keys, scopeNameLabel, scopeVersionLabel, scopeSchemaLabel)
+			kv.vals = append(kv.vals, scopeMetrics.Scope.Name, scopeMetrics.Scope.Version, scopeMetrics.Scope.SchemaURL)
+
+			attrKeys, attrVals := getAttrs(scopeMetrics.Scope.Attributes)
+			for i := range attrKeys {
+				attrKeys[i] = "otel_scope_" + attrKeys[i]
```

Comment:
I was thinking about doing the same for the "downstream" metrics, but I feel that this would be a breaking change.

Reply:
Makes sense.  There isn't really any purpose to the otel_scope_info metric anymore, since it duplicates labels already available on metrics.

Reply:
What about backwards compatibility for OTel Collector Prometheus receiver?

Reply:
We can use an env var to gate the feature and allow migrating from otel_scope_info to otel_scope_<attr>.  We probably shouldn't produce both at once.